src/quo/components/settings/category/style.cljs

```diff
@@ -11,12 +11,13 @@
    :padding-bottom     8})
 
 (defn settings-items
-  [{:keys [label blur? theme]}]
+  [{:keys [label blur? theme background-color-override]}]
```

Comment:
This looks wrong, styles should be encapsulated by the component and set via prop configuration. We should avoid style overrides like this as it becomes a mess. Is it possible you can revisit this solution/ look to encapsulate this in the component.
If you can give me more context I can look into a bit too 👍

Reply:
So I believe this component is the settings-item component under the hood. 
And as such it should be configured like this
<img width="451" alt="Screenshot 2024-03-19 at 22 21 19" src="https://github.com/status-im/status-mobile/assets/22799766/95c8633d-f7f9-46d9-a6b6-0fdbfccf3773">

Reply:
Ultimately it seems like the a design system hack and it would be better if the designers could verify this is right and have the design system reflect this properly. Ideally they don't add overrides like this to the components because it becomes difficult to maintain the design system then. If it's just for this one specific use case it's better they just group the according components, i.e selector and text. 
Can you raise this point with the designers @ibrkhalil ?
cc @Francesca-G

Reply:
Sure, Taking this to the mobile design chat :)

Reply:
Mario is working on fixing this today, Thank you so much Jamie! 🚀

Reply:
After design discussion, I will replace the current component with this variation
https://www.figma.com/file/WQZcp6S0EnzxdTL4taoKDv/Design-System-for-Mobile?type=design&node-id=37048%3A39091&mode=design&t=jGKTGCrTh0cL1bzq-1

Reply:
Can you please check again?